Source/Core/Generic/TokenWrapper.cs

```diff
@@ -14,6 +14,8 @@ public int CompareTo(IToken? other) {
     return Inner.CompareTo(other);
   }
 
+  public bool SourceToken => false;
```

Comment:
```suggestion
  public bool IsSourceToken => false;
```

I initially expected this to be a token rather than a boolean.